Tests/IGListCollectionViewTests.m

```diff
@@ -0,0 +1,65 @@
+//
+//  IGListCollectionViewTests.m
+//  IGListKit
+//
+//  Created by Jeff Bailey on 12/7/16.
+//  Copyright © 2016 Instagram. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <IGListKit/IGListKit.h>
+
+static const CGRect kIGListCollectionViewTestFrame = (CGRect){{0.0, 0.0}, {100.0, 100.0}};
+
+@interface IGListCollectionViewTests : XCTestCase
+
+@property (nonatomic, strong) UIWindow *window;
+@property (nonatomic, strong) IGListCollectionView *collectionView;
+@property (nonatomic, strong) UIViewController *viewController;
+
+@end
+
+@implementation IGListCollectionViewTests
+
+- (void)setUp {
+    [super setUp];
+    self.window = [[UIWindow alloc] initWithFrame:kIGListCollectionViewTestFrame];
+    self.collectionView = [[IGListCollectionView alloc] initWithFrame: kIGListCollectionViewTestFrame collectionViewLayout:[UICollectionViewFlowLayout new]];
+    self.viewController = [UIViewController new];
```

Comment:
If we move in the storyboard tests lets remove all state (properties, setup, teardown) and do it manually per test. That'll make the tests much more localized.